front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -0,0 +1,272 @@
+import React from 'react';
+
+import { Table, TR, TH, TD } from '@ag-media/react-pdf-table';
+import { Page, Text, Image, Document, View, Font } from '@react-pdf/renderer';
+import { useTranslation } from 'react-i18next';
+
+import IBMPlexMonoBold from 'assets/simulationSheet/fonts/IBM-Plex-Mono/IBMPlexMono-Bold.ttf';
+import IBMPlexMonoMedium from 'assets/simulationSheet/fonts/IBM-Plex-Mono/IBMPlexMono-Medium.ttf';
+import IBMPlexMonoRegular from 'assets/simulationSheet/fonts/IBM-Plex-Mono/IBMPlexMono-Regular.ttf';
+import IBMPlexSansBold from 'assets/simulationSheet/fonts/IBM-Plex-Sans/IBMPlexSans-Bold.ttf';
+import IBMPlexSansItalic from 'assets/simulationSheet/fonts/IBM-Plex-Sans/IBMPlexSans-Italic.ttf';
+import IBMPlexSansMedium from 'assets/simulationSheet/fonts/IBM-Plex-Sans/IBMPlexSans-Medium.ttf';
+import IBMPlexSansRegular from 'assets/simulationSheet/fonts/IBM-Plex-Sans/IBMPlexSans-Regular.ttf';
+import IBMPlexSansSemiBold from 'assets/simulationSheet/fonts/IBM-Plex-Sans/IBMPlexSans-SemiBold.ttf';
+import iconAlert from 'assets/simulationSheet/icon_alert_fill.png';
+import logoSNCF from 'assets/simulationSheet/logo_sncf_reseau.png';
+import type { PostStdcmApiResponse, RollingStockWithLiveries } from 'common/api/osrdEditoastApi';
+
+import styles from './SimulationReportStyleSheet';
+import { formatDate, formatDay, getStopDurationTime, getStopTime } from '../utils';
+
+interface SimulationReportSheetProps {
+  stdcmData: PostStdcmApiResponse | null;
+  rollingStockData: RollingStockWithLiveries | undefined;
+  number: string;
+  mapCanvas?: string;
+}
+
+const SimulationReportSheet: React.FC<SimulationReportSheetProps> = ({
+  stdcmData,
+  rollingStockData,
+  number,
+  mapCanvas,
+}) => {
```

Comment:
I think we should avoid adding more React.FC to the codebase for the moment :)
```suggestion
const SimulationReportSheet = ({
  stdcmData,
  rollingStockData,
  number,
  mapCanvas,
}: SimulationReportSheetProps) => {
```

Reply:
done